integration/cluster.go

```diff
@@ -420,7 +420,7 @@ func isMembersEqual(membs []client.Member, wmembs []client.Member) bool {
 
 func newLocalListener(t *testing.T) net.Listener {
 	c := atomic.AddInt64(&localListenCount, 1)
-	addr := fmt.Sprintf("127.0.0.1:%d.%d.sock", c+basePort, os.Getpid())
+	addr := fmt.Sprintf("127.0.0.1:%d%d", c+basePort, os.Getpid())
```

Comment:
can these be 0-padded? otherwise there can be collisions given the right port/pid